tutorial/practicals/nemo/2_nemo_profiling/README.md

```diff
@@ -81,20 +84,25 @@ you have time at the end of this session.)
 
    If you examine the Makefile, you will see that PSyclone has been run with
    the `--profile invokes` option and it is this that causes the subroutine
-   to be instrumented for profiling.
+   to be instrumented for profiling:
+
+       $(PSYCLONE) --profile invokes -api nemo \
+                     -opsy psy.f90 -l output tra_adv_mod.F90
 
 3. Edit the Makefile so that the line invoking PSyclone uses the
    `--profile kernels` option instead. Rebuild (you'll need to do a
```

Comment:
There is 2 "--profile invokes" in the Makefile and only one is relevant for this. Maybe this should be more specific about which Makefile rule to update?

Also the relevant rule has a commented out version without the flag, can this be deleted?

Reply:
Good point - I hadn't quite realised we had so many rules that ran PSyclone in that file. The commented out version is for part 2.1 of the tutorial.